typescript/looker/rtl/browserTransport.ts

```diff
@@ -28,11 +28,30 @@ import {
   ITransportSettings,
   HttpMethod, Authenticator, trace,
   IRequestProps,
-  IRequestHeaders, LookerAppId, agentPrefix,
+  IRequestHeaders, LookerAppId, agentPrefix, ITransport,
 } from './transport'
 import { PassThrough, Readable } from 'readable-stream'
 import { BaseTransport } from './baseTransport'
 import { lookerVersion } from './constants'
+import {ICryptoHash} from "./cryptoHash";
+
```

Comment:
is there a lint rule supporting space inside brackets?  Intellij isn't helping here. `yarn lint` doesn't object.

Reply:
I think VS Code would update on save. IntelliJ .. ask our frontend experts. I've never been able to get it working exactly the way I want.